src/test/analyzer_test.ts

```diff
@@ -338,7 +338,7 @@ suite('Analyzer', () => {
       ],
       sources: ['a.html', 'b.html', 'entrypoint.html'],
     });
-    const analyzer = new BuildAnalyzer(config);
+    const analyzer = new BuildAnalyzer(config, null);
```

Comment:
Why can't we rely on default instead of these explicit nulls?

Reply:
Explicit null is different. It's explicitly saying "no output please".

`undefined` is what you pass in if you want the default.

Reply:
Or to put it another way, why can't it default to null?

Reply:
OH ... sorry I see why.